api/tools/src/main/java/org/pmiops/workbench/tools/CommandLineToolConfig.java

```diff
@@ -104,8 +112,14 @@ GoogleCredential cloudResourceManagerCredentials(CloudStorageService cloudStorag
    */
   @Bean
   @Lazy
+  @Scope("prototype")
```

Comment:
I don't think it needs to block this PR, but I was inspired by your investigation to go down a bit of a Spring rabbit hole - in particular to ensure we have bean isolation across CLI tools. I think that's probably the causative issue this is working around: https://github.com/all-of-us/workbench/pull/2876 . Anyways - this is not ideal but also not terrible as-is in my view.